README.md

```diff
@@ -6,11 +6,10 @@
 
 #### Requirements ####
 
-You will need the following things installed:
- - [NodeJs](https://nodejs.org/en/ "NodeJs")
- - [Go 1.6](https://golang.org/ "Go")
- 
+Requires [Go 1.6](https://golang.org/) to compile and run, along with something
+to compile the SCSS files.
+
 ```bash
-$ ./install.sh
-$ go run main.go
+$ make SASS_COMPILER=sassc # You may use the SASS compiler of your choice
```

Comment:
I would personally prefer that we explicitly tell people to use **only** sassc to ensure that we don't encounter any silly bugs because someone used a different compiler